include/mitsuba/render/mesh.h

```diff
@@ -32,6 +34,12 @@ class MTS_EXPORT_RENDER Mesh : public Shape<Float, Spectrum> {
          Struct *vertex_struct, ScalarSize vertex_count,
          Struct *face_struct,   ScalarSize face_count);
 
+    /// Create a new mesh from a blender mesh
```

Comment:
Should have a docstring including ``\param`` documentation explaining what the various parameters do.